### PR TITLE
Open banking accessibility statement

### PR DESCRIPTION
**Pull request by Akure4Life**

### CODE REVIEW

File: conf/services/open-banking.cy.yml [hunk 1]
```diff
@@ -69,4 +69,5 @@ milestones:
 serviceLastTestedDate: 2021-03-12
 statementVisibility: public
 statementCreatedDate: 2021-03-24
-statementLastUpdatedDate: 2021-03-21
\ No newline at end of file
+statementLastUpdatedDate: 2021-03-24
```
Comment (johnrevill) [inline conf/services/open-banking.cy.yml:72]: This `statementLastUpdatedDate` update looks like it's on the wrong file, shouldn't it have been changed in direct-debit.cy.yml?

File: conf/services/open-banking.yml [hunk 2]
```diff
@@ -61,4 +61,5 @@ milestones:
 serviceLastTestedDate: 2021-03-12
 statementVisibility: public
 statementCreatedDate: 2021-03-24
-statementLastUpdatedDate: 2021-03-21
\ No newline at end of file
+statementLastUpdatedDate: 2021-03-24
```
Comment (johnrevill) [inline conf/services/open-banking.yml:64]: This `statementLastUpdatedDate` update also looks like it's on the wrong file, shouldn't it have been changed in direct-debit.yml?

File: conf/services/open-banking.cy.yml (hunk 1 above)
Comment (Akure4Life) [reply conf/services/open-banking.cy.yml:72]: The date i changed should be on open-banking.cy.yml and

File: conf/services/direct-debit.yml [hunk 3]
```diff
@@ -19,46 +19,46 @@ accessibilityProblems:
...
       On the ‘Verify your email’ page a button is used when displaying the link
       ‘Send it again’. This means the interaction is different to what the user is
       told, either visually or via a screen-reader. This button also does not
       have a focus state which means it is not compatible with keyboard navigation.
       This does not meet WCAG success criterion 2.4.7 (Focus Visible).
-    date: 2020-09-30
+    date: 2021-12-15
 serviceLastTestedDate: 2020-07-06
 statementVisibility: public
 statementCreatedDate: 2020-06-30
```
Comment (johnrevill) [inline conf/services/direct-debit.yml:64]: Does the `statementLastUpdatedDate` need to be updated here?

File: conf/services/direct-debit.cy.yml [hunk 4]
```diff
@@ -22,52 +22,52 @@ milestones:
...
       y cysylltiad ‘Anfonwch eto’. Mae hyn yn golygu bod y rhyngweithio’n
       wahanol i’r hyn a ddywedir wrth y defnyddiwr, naill ai’n weledol neu
       drwy ddarllenydd sgrin. Nid oes gan y botwm hwn nodwedd ffocysu sy’n
       golygu nad yw’n gydnaws â llywio drwy fysellfwrdd. Nid yw hyn yn bodloni
       maen prawf llwyddiant 2.4.7 (Focus Visible) Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
 serviceLastTestedDate: 2020-07-06
 statementVisibility: public
 statementCreatedDate: 2020-06-30
```
Comment (johnrevill) [inline conf/services/direct-debit.cy.yml:73]: Does the `statementLastUpdatedDate` need to be updated here?

File: conf/services/direct-debit.yml (hunk 3 above)
Comment (Akure4Life) [reply conf/services/direct-debit.yml:64]: Yes

File: conf/services/direct-debit.cy.yml (hunk 4 above)
Comment (Akure4Life) [reply conf/services/direct-debit.cy.yml:73]: Yes

**Review by barnesjake — APPROVED** [review]

**Pull request merged**